doc/holochain_101/src/planning_a_dapp.md

```diff
@@ -1 +1,68 @@
 # Planning a dApp
+
+## Introduction
+Generally speaking, you need to know the following in order to build a holochain dApp:
+
+   how to install holochain
+   
+   how to use the command line tools
+   
+   how to configure your application with a "DNA" file
+   
+   how to write your application code in JavaScript or Lisp
+   
+<b>how to think through building a distributed application</b> 
+
+   how to build a user interface for your app
+   
+   how to test your application code
+
+This article will help you plan a dApp by providing practical considerations about the specifics of distributed applications in general, and holochain dApps in particular. It has been remarked that holochain dApps require us to make a mental shift, first from applications whose data is centrally organized, and also from blockchain-based, data-centric dApps.
+
+Here we will provide a basic overview of concepts from cryptography that are central to holochains. 
+Then, we will consider the consequences of holochain's cryptographic architecture for data permissioning, access, and security. 
+
+## What is a dApp? 
```

Comment:
can we move this higher in the page? seems like the very first concept to understand